examples/DesugaringAwait/README.md

```diff
@@ -0,0 +1,54 @@
+# Desugaring !
+
+<details>
+  <summary>What's syntax sugar?</summary>
+  
+  Syntax within a programming language that is designed to make things easier 
+  to read or express. It allows developers to write code in a more concise, readable, or
+  convenient way without adding new functionality to the language itself.
+</details>
+
+Desugaring converts syntax sugar (like `x += 1`) into more fundamental operations (like `x = x + 1`).
```

Comment:
I don't think this is blocking.. but this syntax may not be familiar to roc developers. Maybe there is a better example we could use here?